apps/frontend/src/components/ui/MobileMenu/MobileMenu.module.css

```diff
@@ -0,0 +1,18 @@
+.MobileMenu {
```

Comment:
We are using the camelCase selector naming, please update. The same for the class below